Makefile.am

```diff
@@ -1,9 +1,16 @@
 bin_PROGRAMS = cbridge			
+if ENABLE_TLS
+bin_PROGRAMS += uttun
```

Comment:
No no, this was never even intended to leave my house! :-o

Reply:
Seems useful though, want me to leave it?